packages/client/src/generation/utils/buildNFTAnnotations.ts

```diff
@@ -16,10 +16,14 @@ import { map } from '../../../../../helpers/blaze/map'
  * @returns
  */
 export function buildNFTAnnotations(
```

Comment:
The args for this function are getting lengthy, wdyt about making it an obj?

Reply:
+1

Since the PR needs to be merged ASAP, it's fine to change this later, but please post it to `#project-ts-orm-tech-debt` so it's not lost.

Reply:
https://www.notion.so/prismaio/Technical-Debt-TS-ORM-c199f16035304e2db13793f176098c24?p=78d58e9a82064605890491722be2f0ff